docker/docker-entrypoint.sh

```
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -e
+
+echo "🚀 Starting PDF Merger API..."
+
+# Wait for database to be ready
+echo "⏳ Waiting for database to be ready..."
+until bun run prisma db execute --stdin <<< "SELECT 1" > /dev/null 2>&1; do
```

Shellcheck (0.11.0)

[warning] In POSIX sh, here-strings are undefined.

(SC3011)